addons/website/static/src/components/wysiwyg_adapter/toolbar_patch.js

```diff
@@ -1,15 +0,0 @@
-/** @odoo-module */
-
-import { patch } from "@web/core/utils/patch";
-import { Toolbar } from "@web_editor/js/editor/toolbar";
-
-patch(Toolbar.props, {
-    ...Toolbar.props,
-    showAnimateText: { type: Boolean, optional: true },
-    showTextHighlights: { type: Boolean, optional: true },
-});
-patch(Toolbar.defaultProps, {
-    ...Toolbar.defaultProps,
-    showAnimateText: false,
-    showTextHighlights: false,
-});
```

Comment:
Where did that go?

Reply:
Only reason those were needed is because we were patching the `web_editor.toolbar` template to add those buttons. So we needed a way to hide them outside of website.

Now the buttons are defined in the [WebsiteSnippetsMenu template](https://github.com/odoo-dev/odoo/blob/b39c7a71881096de2beee1f336c66e5a455a9a2d/addons/website/static/src/xml/website.editor.xml#L196) so there is no need for that logic anymore.